doc/base/classes.xml

```diff
@@ -243,7 +243,11 @@
 			<argument index="1" name="y" type="float">
 			</argument>
 			<description>
-				Module (remainder of x/y).
+				Returns the floating-point remainder of x/y (rounded towards zero):
```

Comment:
Small style comment here, if you could fix it with `git commit --amend` and a force push: each newline in the XML is interpreted as a new paragraph in the help, so this will look bad. This would be better:
```
Returns the floating-point remainder of x/y (rounded towards zero):
[codeblock]
fmod = x - tquot * y 
[/codeblock]
Where tquot is the truncated (i.e., rounded towards zero) result of: x/y.
```